commands/daemon.go

```diff
@@ -127,19 +127,25 @@ Note that jobs are not persisted between restarts of the daemon. See
 			Destination: &daemonFlags.repo,
 		},
 		&cli.BoolFlag{
-			Name: "bootstrap",
-			// TODO: usage description
+			Name:        "bootstrap",
+			Usage:       "If set to false the daemon will not connected to peers or sync the chain.",
 			EnvVars:     []string{"VISOR_BOOTSTRAP"},
 			Value:       true,
 			Destination: &daemonFlags.bootstrap,
-			Hidden:      true, // hide until we decide if we want to keep this.
 		},
 		&cli.StringFlag{
 			Name:        "config",
 			Usage:       "Specify path of config file to use.",
 			EnvVars:     []string{"VISOR_CONFIG"},
+			Value:       "~/.lotus/config.toml",
 			Destination: &daemonFlags.config,
 		},
+		&cli.BoolFlag{
+			Name:        "default-config",
+			Usage:       "If true the daemon config file will be overwritten with default configuration values.",
+			Value:       false,
+			Destination: &daemonFlags.commentConfig,
+		},
```

Comment:
nit: I'd personally prefer it if bootstrapping were done in `lily init` to keep concerns tidy, unless there's a good reason to have it here.

Reply:
Agreed. I think the `bootstrap` value would need to be persisted somewhere since it's used when constructing the node:
https://github.com/filecoin-project/lily/blob/2eb534750326b5dead21e94e90259260b32a94ec/commands/daemon.go#L255-L258